grpchantesting/channel_test_cases.go

```diff
@@ -104,8 +104,8 @@ func testUnary(t *testing.T, cli TestServiceClient) {
 	}
 	t.Run("success", func(t *testing.T) {
 		var hdr, tlr metadata.MD
-		req := reqPrototype
-		rsp, err := cli.Unary(ctx, &req, grpc.Header(&hdr), grpc.Trailer(&tlr))
+		req := proto.Clone(&reqPrototype).(*Message)
```

Comment:
These aren't pretty, but these `proto.Clone` calls are here because you can no longer copy a generated message by value -- `go vet` complains since you are implicitly copying a lock :(

Reply:
What a disaster